src/Hooks.tsx

```diff
@@ -50,4 +50,95 @@ function useBrowserStorage<T>(
   return [storedValue, setValue]
 }
 
-export default useBrowserStorage
+// useCookieStorage mirrors useBrowserStorage, but for cookies
+export function useCookieStorage<T>(
+  key: string,
+  initialValue: T,
+  prefix?: string,
+  baseDomain?: string
+): [T, (v: T) => void] {
```

Comment:
A possiblity to minimize changes would be to wrap `useBrowserStorage` and `useCookieStorage` in a common hook called `useStorage`, with a signature looking something like: 
```ts
export function useStorage<T>(
  key: string,
  initialValue: T,
  type: 'session' | 'local' | 'cookie'
  prefix?: string,
  baseDomain?: string
): [T, (v: T) => void] {
  if (type === 'cookie') return useCookieStorage(key, initialValue, prefix, baseDomain)
  else return useBrowserStorage(key, initialValue, type, prefix)
}
```

Then we won't have to move the usage of these hooks out into yet another hook, but can rather just use this directly. It would also make this PR more additive than edit-ive (?), which I like (less code changing is generally good).